chef/cookbooks/crowbar/templates/default/crowbar-post-upgrade.sh.erb

```diff
@@ -66,11 +66,21 @@ if [[ -f /sbin/drbdadm ]] ; then
         log "Waiting for DRBD to bring the devices up..."
         sleep 1
     done
+    drbdadm status
+
+    # Wait for replication to start
+    while ! drbdadm status | grep -qi replication; do
+        log "Replication did not start yet: `drbdadm status`"
+        sleep 1
+    done
+    drbdadm status
+
     # Now wait after DRBD data are synced with the other node
-    while drbdadm status | grep -q Inconsistent; do
+    while drbdadm status | grep -qi -e inconsistent -e replication; do
```

Comment:
Good point. I have to deploy some drbd setup to check how the output should look like